packages/drawer/index.tsx

```diff
@@ -184,15 +190,32 @@ class Drawer extends React.Component<DrawerProps, DrawerState> {
   handleKeyDown = (evt: React.KeyboardEvent<HTMLElement>) => {
     this.props.onKeyDown!(evt);
     if (!this.foundation) return;
-    this.foundation.handleKeydown(evt);
+    this.foundation.handleKeydown(evt.nativeEvent);
   };
 
   handleTransitionEnd = (evt: React.TransitionEvent<HTMLElement>) => {
     this.props.onTransitionEnd!(evt);
     if (!this.foundation) return;
-    this.foundation.handleTransitionEnd(evt);
+    this.foundation.handleTransitionEnd(evt.nativeEvent);
   };
 
+  attachRef = (node: HTMLElement) => {
```

Comment:
I like this pattern - we should actually normalize this across all components. We are doing something similar with textfield but this is a lot cleaner. 
Created an issue:
https://github.com/material-components/material-components-web-react/issues/753